cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/runtime/distributed/remote/RemoteExecutionTwillController.java

```diff
@@ -118,15 +119,18 @@ public void release() {
   public void complete() {
     terminateOnServiceStop = true;
     executionService.stop();
-
+    final RuntimeJobStatus[] status = new RuntimeJobStatus[1];
```

Comment:
shouldn't use an array here, just have a RuntimeJobStatus. Retries.callWithRetries() returns a value, should change it to return the RuntimeJobStatus